dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1068,4 +1061,65 @@ public final Response delete(@Context final HttpServletRequest  request,
         }
     }
 
-}
+    /**
+     * Copies container to the specified host
+     *
+     * @param request
+     * @param response
+     * @param id       id identifier to copy.
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
+    @Path("/{id}/_copy")
+    @JSONP
+    @NoCache
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response copy(@Context final HttpServletRequest request,
```

Comment:
it would be interesting here to think if the user would like or not add a new name in advance

Reply:
This was discussed with @freddyucv and @dsilvam and it was first confirmed then implemented